### PR TITLE
Constrained decoding tutorial

### PR DESCRIPTION
**Pull request by oandreeva-nv**
Constrained decoding part of the tutorial

### CODE REVIEW

File: AI_Agents_Guide/Constrained_Decoding/README.md [hunk 1]
```diff
@@ -31,4 +31,673 @@
...
++ executor_config.logits_post_processor_map = {
++            "<custom_logits_processor_name>": custom_logits_processor
++           }
+self.executor = trtllm.Executor(model_path=...,
+                                model_type=...,
+                                executor_config=executor_config)
+...
+```
+
+Additionally, if you want to enable logits pos-processor for every request
```
Comment (statiraju) [inline AI_Agents_Guide/Constrained_Decoding/README.md:282]: "post processor"

**Review by statiraju — APPROVED** [review]

File: AI_Agents_Guide/Constrained_Decoding/README.md [hunk 2]
```diff
@@ -31,4 +31,673 @@
...
+    }
+  ],
+  "plot_summary": "Harry Potter and the Order of Phoenix is the fifth book in the Harry Potter series. In this installment, Harry returns to Hogwarts School of Witchcraft and Wizardry for his fifth year. The Ministry of Magic is in denial about the return of Lord Voldemort, and Harry finds himself battling against the
+
+```
+
+### Example 2
+
+Optionally, we can also restrict an output to a specific schema. For example,
+in [`client.py`](./artifacts/client.py) we use a `pydentic` library to define the
```
[the remark this answers is not in the document]
Comment (oandreeva-nv) [reply AI_Agents_Guide/Constrained_Decoding/README.md]: updated, thanks for the catch!

File: AI_Agents_Guide/Constrained_Decoding/README.md (hunk 1 above)
Comment (oandreeva-nv) [reply AI_Agents_Guide/Constrained_Decoding/README.md]: thanks for this catch!

**Comment by oandreeva-nv** [comment]
Merging to the main branch [oandreeva_ai_agents_guide](https://github.com/triton-inference-server/tutorials/tree/oandreeva_ai_agents_guide)
